kpatch-build/kpatch-build

```diff
@@ -554,7 +554,10 @@ if [[ $CHANGED -eq 0 ]]; then
 fi
 
 echo -n "Patched objects:"
-for i in "${!objnames[@]}"; do echo -n " $(basename $i)"; done
+for i in $(echo "${objnames[@]}" | tr ' ' '\n' | sort -u | tr '\n' ' ')
```

Comment:
Also "sort -u" only works in the expected way if the input is already sorted.  so might need to "sort | sort -u" for the expected result.